suites/reef/rgw/tier-2_rgw_ms_async_data_notification.yaml

```diff
@@ -364,28 +364,6 @@ tests:
       module: exec.py
       name: set sse-s3 vault configs on multisite
 
-  - test:
-      config:
-        script-name: test_sse_s3_kms_with_vault.py
-        config-file-name: test_sse_s3_per_object.yaml
-        timeout: 300
-      desc: test_sse_s3_per_object
-      module: sanity_rgw_multisite.py
-      name: sse-s3 per object encryption test
-      polarion-id: CEPH-83575153
-
-  - test:
-      clusters:
-        ceph-pri:
-          config:
-            set-env: true
-            script-name: test_sse_s3_kms_with_vault.py
-            config-file-name: test_sse_s3_per_bucket_encryption_normal_object_upload.yaml
```

Comment:
Any reason why this testcase :test_sse_s3_per_bucket_encryption_normal_object_upload  is removed